recipes/plasticparcels/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "plasticparcels" %}
+{% set version = "0.1-alpha1" %}
```

Comment:
This is not pep440 and we cannot publish on conda-forge main unless it complies with CFEP-05.
PS: It is easier, IMO, to tag a stable version.

Reply:
Thanks @ocefpaf. We worked hard today and were able to release a new version, v0.1.0. See updated meta.yaml. Better like this?